lastuser_ui/views/client.py

```diff
@@ -270,8 +270,8 @@ def permission_user_new(client):
         else:
             flash(_(u"Permissions have been assigned to team ‘{pname}’").format(pname=permassign.team.pickername), 'success')
         return render_redirect(url_for('.client_info', key=client.key), code=303)
-    return render_form(form=form, title=_("Assign permissions"), formid='perm_assign',
-        submit=_("Assign permissions"), ajax=True)
+    return make_response(render_template('perms.html', form=form, title=_("Assign permissions"),
+        formid='perm_assign', submit=_("Assign permissions")))
```

Comment:
perms.html shouldn't be necessary after Baseframe takes over widget initialization.